vsphere/datadog_checks/vsphere/config.py

```diff
@@ -94,11 +101,11 @@ def validate_config(self):
                 "disabling ssl verification."
             )
 
-        if self.collection_type not in ('realtime', 'historical', 'both'):
+        if self.collection_type not in ('realtime', 'historical', 'both', 'events_only'):
```

Comment:
General comment, I'm not sure if adding `events_only` as a collection type is the best approach.

Maybe a collect_metrics flag would make the configuration better ?

Reply:
> Maybe a collect_metrics flag would make the configuration better ?

For example?

---

The goal is to allow collecting events only. We need a way to indicate that we don't want to collect historical and realtime metrics.